src/Utilities/useRemediationDataProvider.test.js

```diff
@@ -1,116 +1,133 @@
-/**
- * @jest-environment jsdom
- */
-import { act } from 'react-dom/test-utils';
-import useRemediationDataProvider from './useRemediationDataProvider';
-import {
-} from './Helpers';
-import {
-    fetchViewAdvisoriesSystems,
-    fetchViewSystemsAdvisories
-} from './api';
-
-jest.mock('./api', () => ({
-    ...jest.requireActual('./api'),
-    fetchViewAdvisoriesSystems: jest.fn().mockReturnValue(new Promise((resolve) => {
-        resolve({ data: { testAdvisory: ['test-system'] } });
-    })),
-    fetchViewSystemsAdvisories: jest.fn().mockReturnValue(new Promise((resolve) => {
-        resolve({ data: { testAdvisory: ['test-system'] } });
-    }))
+describe('useRemediationDataProvider', () => it('temproray test', () => {
```

Comment:
Maybe we can use the react testing library to add tests for this hook? https://react-hooks-testing-library.com

Reply:
There are examples of it being used in compliance to test hooks: https://github.com/RedHatInsights/compliance-frontend/blob/master/src/Utilities/hooks/useTableTools/useActionResolver.test.js

Reply:
The issue I faced was more related to mocking the import of a webworker using `import.meta.URL`. There is a library that enables mocking it by configuring babel, but this was causing the webwork not to load into the hook. I am still looking for a way to properly test this.